omigami/plotting.py

```diff
@@ -0,0 +1,60 @@
+from PIL.PngImagePlugin import PngImageFile
+from pandas import DataFrame
```

Comment:
we usually do: `import pandas as pd` and then `pd.DataFrame`. This way we know it's a pandas df and not dask or anything else